README.md

```diff
@@ -4,16 +4,26 @@
 
 Python library that knows how to scare up some config.
 
-# Checks three spots for configuration.
+# Installation
 
-## first
+`pip install git+https://github.com/OAODEV/config-finder.git`
```

Comment:
Might be good to note that the container will need git installed in order to do this.